src/teaching-element/Video.vue

```diff
@@ -1,41 +1,66 @@
 <template>
   <div class="te-video">
-    <plyr>
+    <plyrue ref="plyr" :options="options">
       <video v-if="type.isNative">
         <source :src="url" :type="type.name"/>
       </video>
       <div v-else class="plyr__video-embed">
         <iframe :src="url" allowfullscreen></iframe>
       </div>
-    </plyr>
+    </plyrue>
   </div>
 </template>
 
 <script>
 import { extname } from 'path';
-import { Plyr } from 'vue-plyr';
+import { Plyrue } from 'plyrue';
 
 const NOT_NATIVE = /youtu\.?be|vimeo|drive.google/;
 
 const CUSTOM_TYPE_MAPPING = {
   ogv: 'ogg'
 };
 
+const defaultPlayerOptions = {
+  controls: [
+    'play-large',
+    'play',
+    'progress',
+    'volume',
+    'settings',
+    'fullscreen'
+  ]
+}
+
 export default {
   name: 'te-video',
   props: {
-    url: { type: String, required: true }
+    url: { type: String, required: true },
+    playerOptions: { type: Object, default: () => defaultPlayerOptions }
   },
   computed: {
     type() {
-      if (NOT_NATIVE.test(this.url)) return { isNative: false };
+      const url = new URL(this.url);
+
+      if (url.host === 'drive.google.com' && url.search.includes('export=download')) {
```

Comment:
You mean `url.hostname`. `host` contains both name and port if present.
Also use `url.searchParams.get('export') === 'download'` :bulb: